Editor/MobiledgeXEditorWindow.cs

```diff
@@ -68,6 +68,35 @@ private Texture2D MakeTex(int width, int height, Color col)
         }
 
         private string sdkVersion;
+        private int selectedRegionIndex = 0;
+        private string[] regionOptions = new string[5] { "Nearest", "EU", "JP", "KR", "US" };
+        private string region;
+        public string Region
+        {
+            set { region = value; }
+            get
+            {
+                switch (region)
+                {
+                    case "EU":
+                        return EU_DME;
+                    case "KR":
+                        return KR_DME;
+                    case "JP":
+                        return JP_DME;
+                    case "US":
+                        return US_DME;
+                    case "Nearest":
+                    default:
+                        return "";
```

Comment:
How about pull default/WiFi from the matching engine constant itself?

Reply:
What is the difference ?

Reply:
you have a point, I added wifi.dme it will make it the code look cleaner.